package.json

```diff
@@ -15,7 +15,7 @@
     "testsuite": "STORJ_ALLOW_LOOPBACK=1 NODE_ENV=test ./node_modules/.bin/_mocha --recursive test/**",
     "coverage": "STORJ_ALLOW_LOOPBACK=1 NODE_ENV=test ./node_modules/.bin/istanbul cover ./node_modules/.bin/_mocha -- --recursive",
     "linter": "./node_modules/.bin/eslint ./index.js ./lib ./test",
-    "dev": "NODE_ENV=development nodemon ./bin/storj-billing.js",
+    "start-dev": "NODE_ENV=development nodemon ./bin/storj-billing.js",
```

Comment:
I think it's most recently been changed _to_ `dev`, _from_ `start-dev`.
I agree it should be consistent but perhaps we can go the other way (`start-prod` to `prod`)?

I also think `dev` and `prod` are more conventional.

Reply:
I agree that we should fit it to convention. 

The convention is npm start as start is a special script term. Using that as our default production start command, we would then add an npm run start-dev to start a development instance.

Reference: https://docs.npmjs.com/cli/run-script

Reply:
I also like the convention of doing npm run start:dev instead of using a '-'.

Reply:
Could end up with something like this also which would be quite helpful...

```
"scripts": {
  "start": "if [[ ${NODE_ENV} == \"production\" ]]; then npm run start:prod; else npm run start:dev; fi",
  "start:dev": "webpack",
  "start:prod": "start-cluster"
}
```

Reply:
Yes, `npm start` is a special thing. I think it makes sense to make that what's run for production (`npm start`).

In the development case tho i think `npm run dev` is more conventional than `npm run start-dev`.

Reply:
bridge is also currently using `npm run dev`

Reply:
The convention that I see (note that there is no standard or set convention) is that for scripts or processes, you have a single word (i.e. npm run build). For starting the app, you have either start or start-dev / start:dev. Check the code block above as I think this would be really helpful in kubernetes and docker environments for starting things cleanly and consistently.